management/src/test/java/org/ehcache/management/providers/statistics/HitCountTest.java

```diff
@@ -100,7 +101,14 @@ public void test() throws InterruptedException, IOException {
       registryConfiguration.addConfiguration(new EhcacheStatisticsProviderConfiguration(1,TimeUnit.MINUTES,100,1,TimeUnit.MILLISECONDS,10,TimeUnit.MINUTES));
       ManagementRegistryService managementRegistry = new DefaultManagementRegistryService(registryConfiguration);
 
-      CacheConfiguration<Long, String> cacheConfiguration = CacheConfigurationBuilder.newCacheConfigurationBuilder(Long.class, String.class, resources).build();
+      CacheConfiguration<Long, String> cacheConfiguration = CacheConfigurationBuilder.newCacheConfigurationBuilder(Long.class, String.class, resources)
+        .withEvictionAdvisor(new EvictionAdvisor<Long, String>() {
+          @Override
+          public boolean adviseAgainstEviction(Long key, String value) {
+            return key.equals(2L);
```

Comment:
@ljacomet thanks for the explanation.  A few more quick questions.  In the case of 3 tiers, when there is a hit on the disk tier, that value is only pushed to the heap tier?  It isn't also put in the offheap tier?  I believe that is what you said above, but I just want to double check.  If that is the case then how does an entry ever get into the middle tier(offheap)?  Is it when an entry is evicted from the heap?  I have to work on eviction tests soon, so I need to understand this.  I may have been thinking of it wrong.